TodoListClient/app/routes/s/todo-items/add.js

```diff
@@ -1,19 +1,10 @@
 import Route from '@ember/routing/route';
-import DataRoute from 'ember-data-route';
+import { inject as service } from '@ember/service';
+
+export default class IndexRoute extends Route {
```

Comment:
Should this class be named AddRoute instead?

Reply:
Good catch, I copy pasted the index route and forgot to rename it.